tests/template_tests/filter_tests/test_join.py

```diff
@@ -54,6 +54,24 @@ def test_join08(self):
             "join08", {"a": ["Alpha", "Beta & me"], "var": mark_safe(" & ")}
         )
         self.assertEqual(output, "alpha & beta &amp; me")
+    
+    @setup({"join09": '{{ some_list|join:some_var }}'})
+    def test_join09(self):
+        some_list = ["<p>Hello World!</p>", "beta & me", "<script>Hi!</script>"]
+        some_var = "<br/>"
+        context = {"some_list": some_list, "some_var": some_var}
+        output = self.engine.render_to_string("join09", context)
+        result = '&lt;p&gt;Hello World!&lt;/p&gt;&lt;br/&gt;beta &amp; me&lt;br/&gt;&lt;script&gt;Hi!&lt;/script&gt;'
+        self.assertEqual(output, result)
+    
+    @setup({"join10": '{% autoescape off %}{{ some_list|join:some_var }}{% endautoescape %}'})
+    def test_join10(self):
+        some_list = ["<p>Hello World!</p>", "beta & me", "<script>Hi!</script>"]
+        some_var = "<br/>"
+        context = {"some_list": some_list, "some_var": some_var}
+        output = self.engine.render_to_string("join10", context)
+        result = '<p>Hello World!</p><br/>beta & me<br/><script>Hi!</script>'
+        self.assertEqual(output, result)
```

Comment:
I think we can avoid "some_" prefixes and use more meaningful test method names, e.g.
```suggestion
    @setup(
        {
            "join_autoescape_off": (
                "{% autoescape off %}{{ var_list|join:var_joiner }}{% endautoescape %}"
            )
        }
    )
    def test_join_autoescape_off(self):
        var_list = ["<p>Hello World!</p>", "beta & me", "<script>Hi!</script>"]
        context = {"var_list": some_list, "var_joiner": "<br/>"}
        output = self.engine.render_to_string("join_autoescape_off", context)
        expected_result = "<p>Hello World!</p><br/>beta & me<br/><script>Hi!</script>"
        self.assertEqual(output, expected_result)
```

Reply:
I was copying tests provided in the ticket as such. I will update the code as suggested. Thanks.